lib/elements/fill_stitch.py

```diff
@@ -890,6 +891,12 @@ def underlay_shape(self, shape):
     def fill_shape(self, shape):
         return self.shrink_or_grow_shape(shape, self.expand)
 
+    @property
+    def first_stitch(self):
+        # Serves as a reverence point for the end point of the previous element
+        # This isn't really used for fill stitches as they always make their first stitch point dependent on the previous element itself
+        return None
```

Comment:
Maybe we should return the fill start command position, if the user has attached a fill start command to this fill?

Reply:
Argh, how come did I not think of that?